.github/workflows/e2e.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         globalnet: ['', 'globalnet']
         deploytool: ['operator', 'helm']
-        cable_driver: ['', 'wireguard', 'libreswan']
```

Comment:
Can we squash these into one commit, so we don't have this intermediary step of removing other cable drivers in the git history?